lib/debug/repl.hpp

```diff
@@ -14,6 +14,7 @@
 #include <io/hardware_uart.hpp>
 #include <io/pwm.hpp>
 #include <sensors/accelerometer.hpp>
+#include <sensors/temperature.hpp>
```

Comment:
You also need to include cstdint and optional

Reply:
good point